app/models/concerns/user_access_key_overrides.rb

```diff
@@ -67,6 +69,28 @@ def authenticatable_salt
     ).password_salt
   end
 
+  def generate_password_pii_encryption_key_pair(password)
+    key_pair = generate_pii_encryption_key_pair(password)
+    self.password_encrypted_pii_encryption_key = key_pair.first
+    self.password_pii_encryption_public_key = key_pair.last
```

Comment:
I know this is a proof of concept but here's my pitch for the syntax here if we ever land this
```suggestion
    self.password_encrypted_pii_encryption_key,
      self.password_pii_encryption_public_key = generate_password_pii_encryption_key_pair(password)
```

Reply:
or just do the assignments inside the generate method since it seems to be the same